dev/PushNotifications/PushNotificationsLongRunningTask/NotificationListener.cpp

```diff
@@ -45,14 +50,26 @@ STDMETHODIMP_(HRESULT __stdcall) NotificationListener::OnRawNotificationReceived
 }
 CATCH_RETURN()
 
-const std::string NotificationListener::ConvertProcessNameToUtf8String()
+const std::wstring NotificationListener::GetPayloadAsWideString(unsigned int payloadLength, byte* payload)
 {
-    int size_needed = WideCharToMultiByte(CP_UTF8, 0, m_processName.c_str(), -1, NULL, 0, nullptr, nullptr);
+    int size_needed = MultiByteToWideChar(
```

Comment:
Just call it size since you are reusing this variable twice in the API calls below.